plugin/adsk/plugin/CMakeLists.txt

```diff
@@ -100,6 +100,12 @@ if(IS_MACOSX OR IS_LINUX)
     )
 endif()
 
+# For initializing the handlers of lookdevXUsd
+target_link_libraries(${TARGET_NAME}
+    PRIVATE
+        lookdevXUsd
+)
```

Comment:
This should be wrapped in the cmake flag. We only need to link with it when its built.

Reply:
And you'll need to add a `target_compile_definitions` so you can use ifdef in the plugin.cpp below.